src/redux/boards/actions.js

```diff
@@ -0,0 +1,70 @@
+import * as BoardService from '../../services/BoardService';
+import { stringArrayToObject } from '../../utils/reduxUtils';
+
+export const actions = stringArrayToObject([
+  'GET_BOARDS',
+  'GET_BOARDS_SUCCESS',
+  'GET_BOARDS_FAILURE',
+  'GET_BOARD',
+  'GET_BOARD_SUCCESS',
+  'GET_BOARD_FAILURE'
+]);
+
+const actionCreators = {
```

Comment:
Should we show https://github.com/infinitered/reduxsauce ? ☺️

Reply:
We should! I created this card in trello https://trello.com/c/B970IoDy/23-chequear-redux-sauce